src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,11 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer {
+    @Override
+    public List<Truck> get() {
+        return List.of(new Truck(), new Truck(), new Truck());
```

Comment:
each model of truck or excavator or others should have some sort of a unique field